community/rfcs/24-05-17-OPEA-001-Deployment-Design.md

```diff
@@ -0,0 +1,203 @@
+**Author**
+
+[ftian1](https://github.com/ftian1), [lvliang-intel](https://github.com/lvliang-intel), [hshen14](https://github.com/hshen14), [mkbhanda](https://github.com/mkbhanda), [irisdingbj](https://github.com/irisdingbj), [KfreeZ](https://github.com/kfreez), [zhlsunshine](https://github.com/zhlsunshine) **Edit Here to add your id**
+
+**Status**
+
+Under Review
+
+**Objective**
+
+Have a clear and good design for users to deploy their own GenAI applications on docker or Kubernetes environment.
+
+
+**Motivation**
+
+This RFC presents the OPEA deployment-related design for community discussion.
+
+**Design Proposal**
+
+Refer to this [OPEA overall architecture design document](24-05-16-OPEA-001-Overall-Design.md).
+
+The proposed OPEA deployment workflow is
+
+<a target="_blank" href="opea_deploy_workflow.png">
```

Comment:
Put all the images into the assets folder, and use normal markdown syntax to reference them instead of raw HTML
```suggestion
![OPEA deploy workflow](assets/opea_deploy_workflow.png)
```